cmd/influx/setup.go

```diff
@@ -222,6 +222,10 @@ func onboardingRequest() (*influxdb.OnboardingRequest, error) {
 }
 
 func nonInteractive() (*influxdb.OnboardingRequest, error) {
+	if len(setupFlags.password) < 8 {
```

Comment:
should 8 be defined as a constant that could be changed or configured in the future?